### DISCUSSION TITLE
less secure is not available now in google

### DISCUSSION THREAD

**Discussion by kamat1s** [saran-surya/email_auth#76] [Q&A]
i am having a problem with the test server so i tried to create a custom one but after following all the steps i am getting this error "email-auth >> The remote server is not a valid."

**Comment by saran-surya** [comment]
Hi @kamat1s, 
Yes it is, hope you checked the docs being updated too.
As of now only test servers will work, and production servers will recieve a patch after NOV.

thanks

**Reply by kamat1s** [reply]
Hello @saran-surya thanks for the reply, for now, I changed my code since I'll have to present this tomorrow. the test server is not working for me also I don't know why.

**Reply by saran-surya** [reply]
You have a presentation, give me 15 mins will increase the quotas, you can revert back to the test servers.

**Reply by kamat1s** [reply]
Thank you so much @saran-surya

**Reply by saran-surya** [reply]
You can check the test servers now.

**Reply by saran-surya** [reply]
You can check the test servers now.

**Reply by kamat1s** [reply]
Thank you, i will test it in a bit

**Reply by kamat1s** [reply]
hello @saran-surya I managed to revert my old code but it still outputs the same error. "Unable to access server."

**Reply by kamat1s** [reply]
I figured out a temporary solution, for now, I will wait for the patch for the production server and then try to use the package again. Thank you for being responsive.

**Reply by saran-surya** [reply]
Test servers are back online, Kindly use them. Also check and update regarding the status. Thanks
